src/Locator.php

```diff
@@ -46,12 +50,12 @@ public function getCountryByIP(string $ip)
         return $country;
     }
 
-    public function isValidIp(string $ip)
+    private function isValidIp(string $ip)
     {
-        if (filter_var($ip, FILTER_VALIDATE_IP)) {
-            return true;
+        if (filter_var($ip, FILTER_VALIDATE_IP) === null) {
```

Comment:
According to PHP documentation, the `filter_var` function:

> Returns the filtered data, or FALSE if the filter fails.

Perhaps this needs to check against `false` instead of `null`?

Reply:
Hi @DivineOmega, thank you for your reply. Yes, it should check the value is `false`, not `null`.

![image](https://user-images.githubusercontent.com/9021747/47264711-8b895a00-d54e-11e8-874c-0f60f63c9eec.png)